pkgs/misc/themes/arc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, autoconf, automake, fetchFromGitHub, gnome3, gtk, gtk-engine-murrine, pkgconfig}:
+
+stdenv.mkDerivation rec {
+  version = "f4c71247cf9470037d052ae4a12b86073d0001ff";
```

Comment:
Please fix version according to https://nixos.org/nixpkgs/manual/#sec-package-naming.